sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/PartialMethodNoImplementation.CSharp9.cs

```diff
@@ -0,0 +1,20 @@
+﻿public partial record Record
+{
+    partial void Method(); //Noncompliant {{Supply an implementation for this partial method.}}
+
+    private partial void M2(); // Noncompliant - FP (there is a compiler error already)
```

Comment:
if you also mention `// Error [CSXXXX]` , will you still need `checkMode: CompilationErrorBehavior.Ignore` ?

you can use `// Noncompliant @-1` I think to have the comment on the second line, to have both `Error` and `Noncompliant`

Reply:
Nice!